### PR TITLE
Add repair for gold and silver item.

### PR DESCRIPTION
**Pull request by cainiaowu**
As title says.

Materials | Melting points
-- | --
Gold | 1337.33 K ​(1064.18 °C, ​1947.52 °F)
Silver | 1234.93 K ​(961.78 °C, ​1763.2 °F)
Iron | 1811 K ​(1538 °C, ​2800 °F)

### CODE REVIEW

**Comment by Coolthulhu** [comment]
How much is consumed? The "_small" materials are very granular.

**Comment by cainiaowu** [comment]
most gold/silver things are small jewelry anyway, I just want to add the ability to repair jewelry.

File: data/json/items/tools.json [hunk 1]
```diff
@@ -6661,7 +6661,7 @@
       {
         "type": "repair_item",
         "item_action_type": "repair_metal",
-        "materials": [ "kevlar", "plastic", "iron", "steel", "hardsteel", "aluminum", "copper" ],
+        "materials": [ "kevlar", "plastic", "iron", "steel", "hardsteel", "aluminum", "copper", "silver", "gold" ],
```
Comment (Night-Pryanik) [inline data/json/items/tools.json]: As I pointed in https://github.com/CleverRaven/Cataclysm-DDA/pull/22112#issuecomment-335064701, soldering iron cannot achieve temperature to melt down gold.

Comment (cainiaowu) [reply data/json/items/tools.json]: then It shouldnt repair iron either

Comment (cainiaowu) [reply data/json/items/tools.json]: IMO the repair by soldering iron is to use molten solder "glue" broken parts together.

Comment (Night-Pryanik) [reply data/json/items/tools.json]: > IMO the repair by soldering iron is to use molten solder "glue" broken parts together.

Yes, that's why iron is repairable by soldering iron.

Comment (cainiaowu) [reply data/json/items/tools.json]: I have just checked, you can use solder mixed with brass/silver dust to repair your jewelry. http://m.1688.com/offer/522837469084.html?spm=a26g8.7662792.1998744630.2

Comment (Night-Pryanik) [reply data/json/items/tools.json]: >I have just checked, you can use solder mixed with brass/silver dust to repair your jewelry. 

Jewelry made of what?

Comment (cainiaowu) [reply data/json/items/tools.json]: gold/silver, see the table in this page.
http://www.mining120.com/tech/show-htm-itemid-27135.html

Comment (cainiaowu) [reply data/json/items/tools.json]: Check a bit more, you are right, soldering iron cant use silver solder.

**Pull request merged**